IDEAS/LIDEAS/Components/LinRectangularZoneTemplate.mo

```diff
@@ -0,0 +1,38 @@
+within IDEAS.LIDEAS.Components;
+model LinRectangularZoneTemplate
+  extends IDEAS.Buildings.Components.RectangularZoneTemplate(
+      redeclare replaceable Buildings.Components.ZoneAirModels.WellMixedAir
+      airModel(vol(T(stateSelect=StateSelect.prefer))),redeclare
+      Buildings.Components.InterzonalAirFlow.n50FixedPressure interzonalAirFlow,
+    redeclare LinWindow winA(indexWindow=indexWindowA),
+    redeclare LinWindow winCei(indexWindow=indexWindowCei),
+    redeclare LinWindow winB(indexWindow=indexWindowB),
+    redeclare LinWindow winC(indexWindow=indexWindowC),
+    redeclare LinWindow winD(indexWindow=indexWindowD));
+  parameter Boolean linearise=sim.linearise;
+  parameter Integer indexWindowA=1 if hasWinA "Index of this window A";
+  parameter Integer indexWindowB=1 if hasWinB "Index of this window B";
+  parameter Integer indexWindowC=1 if hasWinC "Index of this window C";
+  parameter Integer indexWindowD=1 if hasWinD "Index of this window D";
```

Comment:
is this correct?

Reply:
The value of the index cannot be guessed upfront as it depends also on the number of zone. It is probably better not to give any default value to it.